crates/control-interface/src/broker.rs

```diff
@@ -1,4 +1,6 @@
 const DEFAULT_TOPIC_PREFIX: &str = "wasmbus.ctl";
+// Copied from https://docs.rs/wasmcloud-core/0.15.0/wasmcloud_core/constant.CTL_API_VERSION_1.html
```

Comment:
Nit: Should probably add some explanation that this is copied to avoid a dep on a crate just for a constant